HW-2/Trie/Trie/Trie.cs

```diff
@@ -0,0 +1,125 @@
+﻿namespace Trie;
+
+/// <summary>
+/// Trie class that implements Trie data structure.
+/// </summary>
+public class Tries : ITrie
+{
+    TrieNode? root;
+
+    public int Size { get; set; }
+
+    public void CreateTrie()
+        => root = new TrieNode();
+
+    public class TrieNode
+    {
+        public Dictionary<char, TrieNode> children = new Dictionary<char, TrieNode>();
+        public bool Word { get; set; }
+        public int WordCounter { get; set; }
+    }
+
+    public bool Add(string element)
+    {
+        if (Contains(element))
+        {
+            return false;
+        }
+        return AddingByRecursion(root, element);
+    }
+
+    bool AddingByRecursion(TrieNode root, string str)
```

Comment:
```suggestion
    bool AddByRecursion(TrieNode root, string str)
```
Методы именуются глаголами в повелительной форме